src/Abp.ZeroCore.EntityFrameworkCore/EntityHistory/EntityHistoryHelper.cs

```diff
@@ -382,6 +404,33 @@ private void UpdateChangeSet(EntityChangeSet changeSet)
                 var entityEntry = entityChange.EntityEntry.As<EntityEntry>();
                 entityChange.EntityId = GetEntityId(entityEntry);
 
+                /* Update audit field */
```

Comment:
```suggestion
                /* Update audit fields */

```

Let's also update this method's summary:
```diff
- /// Updates change time, entity id and foreign keys after SaveChanges is called.
+ /// Updates change time, entity id, audit fields and foreign keys after SaveChanges is called.
```

Reply:
Ryancyq is ready to rewrite this feature, my pr is no longer valid. : )

Reply:
Oops okay!